alcs-api/src/home/home.controller.ts

```diff
@@ -167,4 +183,23 @@ export class HomeController {
     }
     return result;
   }
+
+  private mapAmendmentsToDTOs(amendments: ApplicationAmendment[]) {
```

Comment:
a nitpick 'mapAmendmentsToDtos' so it will match other paces

Reply:
fixed